### ISSUE TITLE
morph_resultsdb overwrites metamorph.json

### ISSUE THREAD

**Issue by gsr-shanks** [RHQE/metamorph#9]
plugins/morph_resultsdb.py when executed overwrites existing metamorph.json instead of appending.

**Comment by Jurisak** [comment]
Solved in #7

**Comment by gsr-shanks** [comment]
It is not solved for morph_resultsdb yet.

**Comment by Jurisak** [comment]
I see it as solved for morph_resultsdb.py. Could you please more describe the problem which you have?

**Comment by gsr-shanks** [comment]
Ok, let me try again

**Comment by gsr-shanks** [comment]
I am unable to query using NVR kernel-pegas-4.10.09.el7, could you please give me another NVR that works ?

**Comment by Jurisak** [comment]
kernel-3.10.0-656.el7

**Comment by gsr-shanks** [comment]
Don't see it anymore.
